src/QuadVelocityController.cpp

```diff
@@ -31,27 +33,33 @@ void QuadVelocityController::setTargetVelocity(geometry_msgs::Twist twist)
 {
     throttle_pid_.setSetpoint(twist.linear.z);
     pitch_pid_.setSetpoint(twist.linear.x);
-    roll_pid_.setSetpoint(twist.linear.y);
+    roll_pid_.setSetpoint(-twist.linear.y);
     yaw_pid_.setSetpoint(twist.angular.z);
 }
 
 // Needs to be called at regular intervals in order to keep catching the latest velocities.
-iarc7_msgs::OrientationThrottleStamped QuadVelocityController::update()
+iarc7_msgs::OrientationThrottleStamped QuadVelocityController::update(const ros::Time& time)
 {
-    // Get the time delta
     geometry_msgs::Vector3 velocities;
     getVelocities(velocities);
-    double time_delta = 0.0;
+
+    // Get the time delta
+    if (last_time_ == ros::Time(0)) {
+        last_time_ = time;
```

Comment:
I think the PID controller should be able to return a value on the first call, using only the P term.  I can think of a couple ways to do that: we could change the PID controller to take a time instead of a delta, just keep track of whether the PID controller has been updated before, or do what I have here.  The upside to what I have here is that it works if the PID controller gets called with the same time after it's been called the first time